src/Tool-Base/PharoShortcuts.class.st

```diff
@@ -55,7 +55,7 @@ PharoShortcuts >> backwardLineShortcut [
 { #category : #'keymaps - editor edit' }
 PharoShortcuts >> backwardWordDeleteShortcut [
 
-	^ Character backspace shift
+	^ Character backspace ctrl
```

Comment:
Good for Windows. Not sure about Mac support.

Reply:
Changed + Added comment